translations/en.json

```diff
@@ -67,9 +67,12 @@
   "contributers": "Contributors",
   "thanks_translators": "A special thanks to our translators: ",
   "thanks_coders": "A special thanks to our coders: ",
-  "instance_totals": "You can access all content in the lemmyverse from any server/instance. The lemmyverse currently has {{instances}} instances, and {{users}} monthly active users.",
-  "instance_comparison": "For a more detailed comparison of Lemmy instances, see",
-  "instance_browser": "After you create an account, you can find communites across all instances using",
+  "instance_explanation": "<1>To use Lemmy, you need to be a member of <1>one instance</1>. You will still be able to see content from anywhere, but the instance you choose will determine:</1><2><1>What URL you use to log in to Lemmy,</1><1>What content shows on the homepage when you select \"Local\" or \"All\",</1><1>Who moderates your instance, and</1><1>What rules you agree to when you sign up.</1></2>",
+  "choose_instance": "Choose an instance that matches your interests, language, and region. <1>Please avoid joining instances that are already crowded (1K+ users/month).</1> If an instance gets overcrowded, it can start running slowly or experiencing downtime, so choosing an uncrowded instance will give both you and others a better Lemmy experience.",
```

Comment:
This should be changed as users per month number is removed https://github.com/LemmyNet/joinlemmy-site/pull/173/files